Jellyfin.Server.Implementations/FullSystemBackup/BackupService.cs

```
@@ -0,0 +1,351 @@
+            CopyDirectory(Path.Combine(_applicationPaths.DataPath, "ScheduledTasks"), Path.Combine("Data", "ScheduledTasks"));
+            CopyDirectory(Path.Combine(_applicationPaths.DataPath, "subtitles"), Path.Combine("Data", "subtitles"));
+            CopyDirectory(Path.Combine(_applicationPaths.DataPath, "trickplay"), Path.Combine("Data", "trickplay"));
+
+            using var manifestEntry = zipArchive.CreateEntry(_manifestEntryName).Open();
+            await JsonSerializer.SerializeAsync(manifestEntry, manifest).ConfigureAwait(false);
+        }
+
+        _logger.LogInformation("Backup created");
+    }
+
+    /// <inheritdoc/>
+    public async Task<BackupManifestDto[]> EnumerateBackups()
+    {
+        var archives = Directory.EnumerateFiles(_applicationPaths.BackupPath, "*.zip");
+        var manifests = new List<BackupManifestDto>();
+        foreach (var item in archives)
+        {
+            try
+            {
+                using var archiveStream = File.OpenRead(item);
+                using var zipStream = new ZipArchive(archiveStream, ZipArchiveMode.Read);
+                var manifestEntry = zipStream.GetEntry(_manifestEntryName);
+                if (manifestEntry is null)
+                {
+                    continue;
+                }
+
+                using var manifestStream = manifestEntry.Open();
+                var manifest = await JsonSerializer.DeserializeAsync<BackupManifest>(manifestStream, _serializerSettings).ConfigureAwait(false);
+
+                if (manifest is null)
+                {
+                    continue;
+                }
+
+                manifests.Add(new BackupManifestDto()
+                {
+                    BackupEngineVersion = manifest.BackupEngineVersion,
+                    DateOfCreation = manifest.DateOfCreation,
+                    JellyfinVersion = manifest.JellyfinVersion
+                });
+            }
+            catch
+            {
+                continue;
```

Shouldn't silently ignore errors